sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_patch.py

```diff
@@ -6,13 +6,34 @@
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
 import base64
-from typing import List, overload, Union, Any, Optional
+import json
+import sys
+from typing import Any, Callable, Dict, IO, List, Optional, TypeVar, Union, overload
+
+from azure.core.exceptions import ClientAuthenticationError, HttpResponseError, ResourceExistsError, ResourceNotFoundError, ResourceNotModifiedError, map_error
+from azure.core.exceptions import HttpResponseError
 from azure.core.messaging import CloudEvent
 from azure.core.tracing.decorator import distributed_trace
 from ._operations import EventGridClientOperationsMixin as OperationsMixin
+from .._model_base import AzureJSONEncoder
 from ..models._models import CloudEvent as InternalCloudEvent
 from ..models._patch import ReceiveResult, ReceiveDetails
+from azure.core.pipeline import PipelineResponse
+from azure.core.rest import HttpRequest, HttpResponse
+from azure.core.utils import case_insensitive_dict
```

Comment:
I think you'll have to move these imports above the local ones to where the other core imports are